client/lib/plans/constants.js

```diff
@@ -154,6 +154,13 @@ export const FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED_SIGNUP = 'unlimited-backup';
 export const FEATURE_MEMBERSHIPS = 'memberships';
 export const FEATURE_PREMIUM_CONTENT_BLOCK = 'premium-content-block';
 
+// jetpack features category
+export const FEATURE_CATEGORY_SECURITY = Symbol();
```

Comment:
Symbols are guaranteed to be unique, whereas strings can be duplicated inadvertently. The polyfill allows this basic usage.